### ISSUE TITLE
Search option for multi-selection

### ISSUE THREAD

**Issue by edrock200** [Sonarr/Sonarr#6210]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Perhaps I'm missing it and I attempted to search but didn't see it, apologies if this is a dupe, but when doing a bulk edit of shows, there doesn't appear to be a way to kick off a search for selected shows.

### Describe the solution you'd like

A search option to search for monitored episodes when selecting multiple shows

### Describe alternatives you've considered

Click each show then search.

**Comment by markus101** [comment]
Duplicate of https://github.com/Sonarr/Sonarr/issues/5679

**Comment by edrock200** [comment]
Apologies for the dupe

**Comment by edrock200** [comment]
@markus101 apologies if I'm being dumb - in the other thread you mentioned missing and cutoff unmet searches. I don't see a way to do this either unless you mean the wanted/search all function? I don't have a canned cutoff unmet filter, I do have a missing. Here's a use case to possible consider the bulk search option. Recently lost a drive in a jbod setup. This created a bunch of random "holes" in various series where a handful of episodes are missing. First, I refreshed all to see the missing missing episodes. Then I sorted I filtered by shows that have 90+% episode completion, selected all, and changed the monitoring to "missing." This is where that search option would be great. I could select series in batches and search vs one by one.

Understand if you still feel its not of value but would make recovery scenarios like this much easier to deal with unless there's an easier way I'm not thinking of.
